src/sonic-config-engine/openconfig_acl.py

```diff
@@ -32,14 +32,10 @@ class yc_state_openconfig_acl__acl_state(PybindBase):
   """
   __slots__ = ('_pybind_generated_by', '_path_helper', '_yang_name', '_extmethods', '__counter_capability',)
 
-  _yang_name = 'state'
-
-  _pybind_generated_by = 'container'
-
   def __init__(self, *args, **kwargs):
-
+    self._yang_name = 'state'
+    self._pybind_generated_by = 'container'
```

Comment:
is this generated use the command on line 13?

Reply:
No. I did not regenerate the file, I simply fixed Python 3 errors, since we currently need this file to be compatible with both Python 2 and 3. Once we remove support for Python 2 completely, we can regenerate the file using Python 3.